rtf_vis_tool/src/Components/Data_Association_PC.js

```diff
@@ -0,0 +1,113 @@
+import React, {useEffect, useState, useRef} from "react";
+import {Canvas, extend} from "react-three-fiber";
+import {OrbitControls} from 'three/examples/jsm/controls/OrbitControls';
+import '../stylesheets/Data_Association_PC.css';
+
+//Loading Components
+import CoordinateGrid from './CoordinateGrid';
+import OrbitControlsComponent from './OrbitControlsComponent';
+import PointMesh from './PointMesh';
+
+//allows user to orbit and pan around the point cloud
+extend({OrbitControls})
+
+//Point Cloud Renderer Component
+//Spawned once the 'SfMData' node is clicked
+const Data_Association_PC = (props) => {
+    const [pointCloud, setPointCloud] = useState([]);
+    const [isCentered, setIsCentered] = useState(false);
+    const [showCoordGrid, setShowCoordGrid] = useState(true);
+    const lightgray = `rgb(163, 168, 165)`;
+    const pointSizeArr = [0.2,8,8];
```

Comment:
what do 0.2 and 8 represent?

Reply:
0.2 represents the radius of the points and 8s represent something called height and width segments. The 8s are actually optional parameters, so I have just left the 0.2 to keep it simple. Added a comment in the file as well about it.